include/vrb/Matrix.h

```diff
@@ -91,6 +91,25 @@ class Matrix {
     return result;
   }
 
+  static Matrix Rotation(const Vector& aDirection, const Vector& aUp = vrb::Vector(0.0f, 1.0f, 0.0f)) {
```

Comment:
I think you want to normalize the aDirection to be safe?

Reply:
yes, it seems safer. I updated the code.